R/refmodel.R

```diff
@@ -459,12 +463,6 @@ init_refmodel <- function(object, data, formula, family, ref_predfun = NULL,
   target <- .get_standard_y(y, weights, family)
   y <- target$y
 
-  if (inherits(object, "brmsfit")) {
-    object$elapsed_time <- rstan::get_elapsed_time(object$fit)
```

Comment:
We can keep this here until brms returns `object$elapsed_time`.

Reply:
Ok. We can add it in the `get_refmodel.brmsfit`.

Do you wanna keep this `brmsfit` check: https://github.com/hsbadr/projpred/blob/39b5321e74f571b8e458316652d56588632de964/R/cv_varsel.R#L803

Reply:
https://github.com/paul-buerkner/brms/pull/1136/commits/2b83b4ee7b3aa6d267c1042d915f0b13303382db should do it.

Reply:
We can wait to merge this until that merge goes through on brms.

Reply:
Sounds good.